webapp/components/integrations/components/installed_incoming_webhooks.jsx

```diff
@@ -111,6 +111,7 @@ export default class InstalledIncomingWebhooks extends React.Component {
                     creator={this.state.users[incomingWebhook.user_id] || {}}
                     canChange={canChange}
                     team={this.props.team}
+                    channel={channel}
```

Comment:
If `InstalledIncomingWebhooks` already has a dependency on the channel then the container can be pushed to that level instead of the `InstalledIncomingWebhook` level and should be included as a prop there like it already is in your PR. Basically if something affects rendering we should move it into the props. In this case if the channel affects the rendering of `InstalledIncomingWebhooks` it should be in its props, and the way do that with Redux when it's not passed from the parent is using a container like our `index.js` files

Does that make sense?